Dfe.Data.SearchPrototype/Web/Tests/Integration/PageObjectModel/PageComponents/PageHeader.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using DfE.Data.SearchPrototype.Web.Tests.Integration.PageObjectModel.Setup;
+using Microsoft.AspNetCore.Mvc.Testing;
+
+namespace DfE.Data.SearchPrototype.Web.Tests.Integration.PageObjectModel.PageComponents
+{
+    public class PageHeader : PageObjectModelExtractor
+    {
+        public PageHeader(WebApplicationFactory<Program> webApplicationFactory) :
+            base(webApplicationFactory){
+        }
+
+        public async Task<string> GetHeading()
+        {
+            IElement? searchHeading = await GetSearchHeaderElement("h1");
+
+            return searchHeading == null ?
+                throw new InvalidOperationException("Unable to derive the search heading.") :
+                searchHeading.InnerHtml;
+        }
+
+        public async Task<IHtmlAnchorElement> GetSearchHeaderLink(string linkName)
```

Comment:
would this be a header within Search only? we could do it as GetHeaderLink and then check it within all pages

Reply:
For use across all pages potentially, but it's there really to demo the principle here. This could be renamed going forward and expanded with more behaviour as we deem it relevant.